Modules/_testsinglephase.c

```diff
@@ -140,7 +140,9 @@ init_module(PyObject *module, module_state *state)
     if (initialized == NULL) {
         return -1;
     }
-    if (PyModule_AddObjectRef(module, "_module_initialized", initialized) != 0) {
+    int rc = PyModule_AddObjectRef(module, "_module_initialized", initialized);
+    Py_DECREF(initialized);
+    if (rc != 0) {
```

Comment:
Please write idiomatic C code:

```suggestion
    if (rc < 0) {
```